### ISSUE TITLE
Make the story form not use as much JavaScript

### ISSUE THREAD

**Issue by connorshea** [noidedmedia/Fiction-Dock#21]
:cry: think of the children :cry:

**Comment by AnthonySuper** [comment]
It already uses the minimal possible amount of JavaScript, I think. Renders the fields for just the story inline, uses JS for relations.

**Comment by connorshea** [comment]
Let me rephrase:

I don't want to have to write JavaScript to make the layout work with the generic form CSS I've been using, because JavaScript is the devil.

**Comment by connorshea** [comment]
We tried.
